### PR TITLE
FOUR-19989 : The cases table overlays recent searches.

### PR DESCRIPTION
**Pull request by henryjonathanquispe**
## Issue & Reproduction Steps
1.Log In
2.Perform some searches
3.Go to cases tab
4.Click on AI search

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-19989

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy

### CODE REVIEW

File: resources/jscomposition/base/table/BaseTable.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
       :class="{
         'tw-table-fixed':getDefaultConfig(config).tableFixed
       }">
-      <thead class="tw-border-b tw-sticky tw-top-0 tw-z-10 tw-bg-gray-100">
+      <thead class="tw-border-b tw-sticky tw-top-0 tw-z-[9] tw-bg-gray-100">
```
Comment (fagubla) [inline resources/jscomposition/base/table/BaseTable.vue:10]: Isn't it better to remove the z-index?

Comment (henryjonathanquispe) [reply resources/jscomposition/base/table/BaseTable.vue:10]: its neccesary for the avatar column

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-862fd55d82.engk8s.processmaker.net

**Comment by pmPaulis** [comment]
This is working as expected
Captura de pantalla 2024-11-05 a las 1 10 10 p  m

**Review by pmPaulis — APPROVED** [review]

**Review by fagubla — APPROVED** [review]

**Pull request merged**